.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "libs/mavlink/include/mavlink/v1.0"]
 	path = libs/mavlink/include/mavlink/v1.0
-	url = https://github.com/mavlink/c_library_v1.git
+	url = https://github.com/jgoppert/c_library_v1.git
```

Comment:
I switched this over temporarily to build the new messages for me on a separate repo.